### DISCUSSION TITLE
Working in Monogame with 3D has never been easier, and Bepu V2 is great!

### DISCUSSION THREAD

**Discussion by arismoko** [bepu/bepuphysics2#203] [Show and tell]
Working on a 3D application framework right now, and I'd just like to sincerely say thank all the people who contributed to this, and Ross. I've been using Unity for the past few years, and out of a desire to learn I found Monogame, and subsequently Bepu. I read online that 3D can be frustrating when using Mono, and while that may be true for beginners, Bepu Physics/Utilities has helped me immensely.

Incredible performance, I'm building a wrapper for bepu that manages the physics automatically, and even with the added overhead I haven't seen my processor break a sweat! (: 👍

**Comment by RossNordby** [comment]
Thanks, glad it's working for you!
